clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/description/servergroup/DeployDcosServerGroupDescription.groovy

```diff
@@ -73,19 +79,15 @@ class DeployDcosServerGroupDescription extends AbstractDcosCredentialsDescriptio
   @Canonical
   static class Docker {
     Image image
-    NetworkType network
+
+    // TODO this isn't actually used
+    String network
```

Comment:
I need to address these various TODOs around this network field. It exists in various places, but really in DC/OS only applies in the container/docker section. If there is no container/docker, then the network type is by default HOST in DC/OS